terra/templates/terra/employee.html

```diff
@@ -31,11 +31,13 @@ <h2>FY {{fiscal_year}} Travel Requests</h2>
                 <th scope="col">Approved Funds</th>
                 <th scope="col">Actual Expenses</th>
                 <th scope="col">Days Out Working</th>
-                <th scope="col">Days Out Vacation</th>   
+                <th scope="col">Days Out Vacation</th>
+                <th scope="col">Days Out Total</th>    
             </tr>
         </thead>
         <tbody>
     {% for treq in employee.travelrequest_set.all %}
+    {% if treq.in_fiscal_year %}
```

Comment:
Minor nit: indentation, this `if` should be inside the `for`, similar to line 58-59.